source/digits_hits/include/GateJPETActorMessenger.hh

```diff
@@ -41,6 +56,7 @@ protected:
 	G4UIcmdWithABool* pEnableProcessNameCmd;
 	G4UIcmdWithABool* pEnableEmissionPointCmd;
 	G4UIcmdWithABool* pEnablePrimaryEnergyCmd;
+	G4UIcmdWithAString* pEnableASCIFileNameCmd;
```

Comment:
Is is initialized in the constructor?

Reply:
Nope - becouse whole pointers are initialized in *BuildCommands* - all messanger classes are created in this way in Gate